package/tsconfig.json

```diff
@@ -3,15 +3,16 @@
     "include": ["./src/**/*"],
     "compilerOptions": {
         "rootDir": "./src",
-        "module": "amd",
+        "module": "esnext",
```

Comment:
es5 please

Reply:
This is just used for type-checking now. It's still getting transpiled, the config now lives in /package/scripts/rollup.amd.js